src/vs/workbench/contrib/inlineChat/browser/inlineChatStrategies.ts

```diff
@@ -577,7 +577,7 @@ export class LiveStrategy extends EditModeStrategy {
 			message = localize('change.0', "Nothing changed.");
 		} else if (remaining === 1) {
 			message = needsReview
-				? localize('review.1', "Accept or Discard change")
+				? localize('review.1', "$(info) Accept or Discard change")
```

Comment:
Prevent localizers from making mistakes:

```suggestion
				? `$(info) ${localize('review.1', "Accept or Discard change")}`
```